### ISSUE TITLE
Undefined symbol

### ISSUE THREAD

**Issue by bonh** [ibaned/omega_h#255]
Recent master branch with fenics 2018.1.0 docker image. Compilation works fine. 

`import PyOmega_h as omega_h`

results in 
`ImportError: /usr/local/lib/python3/dist-packages/PyOmega_h.cpython-35m-x86_64-linux-gnu.so: undefined symbol: _ZN7Omega_h10pybind11_cERN8pybind116moduleE`

Am I missing something?

Tried with develop branch, problem persists.

Using `nm` and demangle the symbol gives
 `Omega_h::pybind11_c(pybind11::module&)`

**Comment by bonh** [comment]
Compiling master with
```
diff --git a/src/PyOmega_h.cpp b/src/PyOmega_h.cpp
index 18cc539e..8b8f01cb 100644
--- a/src/PyOmega_h.cpp
+++ b/src/PyOmega_h.cpp
@@ -2,7 +2,7 @@
 
 PYBIND11_MODULE(PyOmega_h, module) {
   module.doc() = "Omega_h: simplex mesh adaptation";
-  Omega_h::pybind11_c(module);
+  //Omega_h::pybind11_c(module);
   Omega_h::pybind11_array(module);
   Omega_h::pybind11_comm(module);
   Omega_h::pybind11_library(module);
diff --git a/src/PyOmega_h.hpp b/src/PyOmega_h.hpp
index d0845ae5..ec3203da 100644
--- a/src/PyOmega_h.hpp
+++ b/src/PyOmega_h.hpp
@@ -19,7 +19,7 @@ namespace py = pybind11;
 namespace Omega_h {
 class Library;
 extern std::unique_ptr<Library> pybind11_global_library;
-void pybind11_c(py::module& module);
+//void pybind11_c(py::module& module);
 void pybind11_array(py::module& module);
 void pybind11_comm(py::module& module);
 void pybind11_library(py::module& module);
```
and `python3 -c "import PyOmega_h"`gives 
`File "<string>", line 1, in <module>
ImportError: arg(): could not convert default argument into a Python object (type not registered yet?). Compile in debug mode for more information.`

**Comment by ibaned** [comment]
This is strange... I thought I had been checking the FEniCS build every release... I'm looking into it

**Comment by ibaned** [comment]
Ah, this is only caught when importing... I'll remember to do an actual import before releasing next time

**Comment by ibaned** [comment]
This should be fixed in the latest release, `v9.17.1`. I also will now test importing before future releases.

**Comment by bonh** [comment]
Thy, will test and report
